core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -293,15 +298,13 @@ void SYCLSpaceInitializer::fence() {
 }
 
 void SYCLSpaceInitializer::print_configuration(std::ostream& msg,
-                                               const bool /*detail*/) {
+                                               const bool detail) {
   msg << "Devices:" << std::endl;
   msg << "  KOKKOS_ENABLE_SYCL: ";
   msg << "yes" << std::endl;
```

Comment:
This information is duplicated

Reply:
I just tried to be consistent with, say, `SYCL`:
```
$ git grep -n "KOKKOS_ENABLE_HIP" | grep '"'
Makefile.kokkos:1168:    tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE")
core/src/HIP/Kokkos_HIP_Instance.cpp:105:  s << "macro  KOKKOS_ENABLE_HIP : defined" << '\n';
core/src/HIP/Kokkos_HIP_Space.cpp:496:  msg << "  KOKKOS_ENABLE_HIP: ";
core/src/HIP/Kokkos_HIP_Space.cpp:500:  msg << "  KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE: ";
```
and `CUDA`:
```
$ git grep -n "KOKKOS_ENABLE_CUDA" | grep '"' | grep "<<"
core/src/Cuda/Kokkos_Cuda_Instance.cpp:252:  s << "macro  KOKKOS_ENABLE_CUDA      : defined\n";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:885:  msg << "  KOKKOS_ENABLE_CUDA: yes\n";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:888:  msg << "  KOKKOS_ENABLE_CUDA_ATOMICS: ";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:896:  msg << "  KOKKOS_ENABLE_CUDA_LAMBDA: ";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:902:  msg << "  KOKKOS_ENABLE_CUDA_LDG_INTRINSIC: ";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:908:  msg << "  KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE: ";
core/src/Cuda/Kokkos_Cuda_Instance.cpp:914:  msg << "  KOKKOS_ENABLE_CUDA_UVM: ";
```

What do you want me to do?

Reply:
My preference would go towards dropping the macro name in `*SpaceInitializer::print_configuration` for these backends.